src/UCTSearch.h

```diff
@@ -28,6 +28,7 @@
 #include "GameState.h"
 #include "KoState.h"
 #include "UCTNode.h"
+#include "../gtest/googletest/include/gtest/gtest_prod.h"
```

Comment:
Ugh no, definitely don't add a GTest dependency into the normal codebase itself. I'd rather add a getter or a function that returns the maximum amount of time a search will run.